app/Http/Requests/ImageUploadRequest.php

```diff
@@ -42,46 +43,65 @@ public function response(array $errors)
      * @param String $path  location for uploaded images, defaults to uploads/plural of item type.
      * @return SnipeModel        Target asset is being checked out to.
      */
-    public function handleImages($item, $w = 550, $path = null)
+    public function handleImages($item, $w = 550, $fieldname = 'image', $path = null)
```

Comment:
There was always the option of changing the field name on the `users` table to `image` to avoid this extra complexity, but I imagine that would break stuff in other places.